src/profile.cpp

```diff
@@ -1073,7 +1073,7 @@ void Profile::storeSettingsByArray(const QString &prefix, const QList<QVariantMa
 
 bool Profile::forcedLibrariesEnabled(const QString &executable) const
```

Comment:
Since you move these two to uibase you should probably add `override` in the declaration (in the header).

Reply:
These are no longer being moved